edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesPodBuilder.cs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using k8s.Models;
+    using Microsoft.Azure.Devices.Edge.Agent.Core;
+    using Microsoft.Azure.Devices.Edge.Agent.Docker.Models;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using AgentDocker = Microsoft.Azure.Devices.Edge.Agent.Docker;
+
+    public class KubernetesPodBuilder
+    {
+        const string SocketDir = "/var/run/iotedge";
+        const string ConfigVolumeName = "config-volume";
+        const string TrustBundleVolumeName = "trust-bundle-volume";
```

Comment:
Shouldn't we use values that came from ENV?

Reply:
Yes, we should,

Reply:
Addressed this in another review